### PR TITLE
Pegasus pruning - Code.org marketing site - Summer 2022

### PR DESCRIPTION
**Pull request by kelbyhawn**
Remove old, irrelevant, and draft pages from the `pegasus/sites.v3/code.org/public/` directory which runs the [Code.org](https://code.org) marketing site. 

**Relevant info:** [Asana task](https://app.asana.com/0/1202773141567416/1202778825975827/f) • [Site Audit Gsheet](https://docs.google.com/spreadsheets/d/1pRkNtthNeddMO_dsXRcb59BuidSezAHojjIMwfw-_PQ/edit#gid=0) • [Google Drive folder w/ copies of old announcement pages](https://drive.google.com/drive/folders/1541wmzO-ZSVgZfodeAA0ETTRkuQV0TZy) 

**Related PRs:** https://github.com/code-dot-org/code-dot-org/pull/47425 • https://github.com/code-dot-org/code-dot-org/pull/47721 • https://github.com/code-dot-org/code-dot-org/pull/48702

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202778825975832
  - https://app.asana.com/0/0/1202778825975827

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/certificates/splat.haml
Comment (davidsbailey) [inline pegasus/sites.v3/code.org/public/certificates/splat.haml]: this file is still needed. once swarms are over I am working on moving this functionality to dashboard, after which I'll be able to remove this.

File: pegasus/sites.v3/code.org/public/markdown/advanced.moved
Comment (davidsbailey) [inline pegasus/sites.v3/code.org/public/markdown/advanced.moved]: I think we need to keep the files in this directory, because of existing code references: https://github.com/code-dot-org/code-dot-org/blob/4449f68613ddcfcf79f864abad873bd8fc515ea6/pegasus/sites.v3/code.org/views/markdown_nav.haml#L4-L9

File: pegasus/sites.v3/csedweek.org/public/about/index.md
Comment (davidsbailey) [inline pegasus/sites.v3/csedweek.org/public/about/index.md]: what indication do we have that csedweek.org is no longer being used? I could be missing something but it looks to me like we might not be tracking this entire domain name with GA.

File: pegasus/sites.v3/code.org/public/page_mode.haml
Comment (davidsbailey) [inline pegasus/sites.v3/code.org/public/page_mode.haml:3]: this looks like a file that engineers use to quickly test what the page mode is.  @breville , maybe you can confirm whether it is needed?

**Review by davidsbailey — COMMENTED** [review]
thank you so much for working on this Kelby! I having finished reviewing, but wanted to share some initial comments.

File: pegasus/sites.v3/code.org/public/page_mode.haml
Comment (breville) [reply pegasus/sites.v3/code.org/public/page_mode.haml:3]: I don't think it's been used for a long time, so it's alright to remove.

File: pegasus/sites.v3/csedweek.org/public/about/index.md
Comment (davidsbailey) [reply pegasus/sites.v3/csedweek.org/public/about/index.md]: I talked with Brendan, and it sounds like although csedweek.org is still a live web site, it is no longer hosted by us. so, that confirms that this directory is safe to delete.

based on that, my request would be to split the removal of the csedweek.org directory off into a separate PR, since this PR is getting quite big, and keeping it all together will add to the difficulty if we break something and have to undo some of this work. sorry for the added work, but I do think it'll give us a bit more confidence in shipping all of this.

File: pegasus/sites.v3/code.org/public/certificates/splat.haml
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/certificates/splat.haml]: Reinstated ✅

File: pegasus/sites.v3/code.org/public/markdown/advanced.moved
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/markdown/advanced.moved]: Reinstated ✅

File: pegasus/sites.v3/code.org/public/page_mode.haml
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/page_mode.haml:3]: Sounds good, will resolve this convo.

File: pegasus/sites.v3/csedweek.org/public/about/index.md
Comment (kelbyhawn) [reply pegasus/sites.v3/csedweek.org/public/about/index.md]: This works for me! I'll reinstate this directory and make a new PR. https://github.com/code-dot-org/code-dot-org/pull/47425

File: pegasus/sites.v3/code.org/public/csd/standards.moved
Comment (davidsbailey) [inline pegasus/sites.v3/code.org/public/csd/standards.moved]: please reinstate this one, because we don't have a good way to tell if it is being used (GA doesn't log anything for redirects like this)

**Review by davidsbailey — APPROVED** [review]
Thank you for doing this @kelbyhawn ! This is going to make life a lot easier for many people :-) I'd suggest waiting for Brendan's approval too, since he likely has more context than I do on many of these files.

LGTM after addressing 1 comment below.

**Review by breville — APPROVED** [review]

File: pegasus/sites.v3/code.org/public/csd/standards.moved
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/csd/standards.moved]: @davidsbailey done!

**Pull request merged**
